Src/Support/Google.Apis.Auth.AspNetCore3/Google.Apis.Auth.AspNetCore3.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\CommonProjectProperties.xml" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Version>$(Version)-beta01</Version>
+  </PropertyGroup>
+
+  <!-- nupkg information -->
+  <PropertyGroup>
+    <Title>Google APIs ASP.NET Core Auth Extensions</Title>
+    <Description>
+The Google APIs Client Library is a runtime client for working with Google Services.
+
+The ASP.NET Core Auth extension library contains a Google-specific OpenIdConnect auth handler.
+Supports incremental auth, and an injectable IGoogleAuthProvider to supply Google credentials.
+
+Supported Platforms:
+- ASP.NET Core 3.0+
+    </Description>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\Google.Apis.Auth.AspNetCore\Consts.cs" Link="Consts.cs" />
```

Comment:
What happens if you just have `<Compile Include="..\Google.Apis.Auth.AspNetCore\*.cs" />`? Does that work? If so, it might be simpler than listing all the files separately.

Reply:
Done, it appears to work fine :)